tests/phpunit/tests/rest-api/rest-attachments-controller.php

```diff
@@ -217,6 +217,7 @@ public function test_registered_query_params() {
 			'video',
 			'image',
 			'audio',
+			'font',
```

Comment:
This change is causing the tests to fail. 

Possibly because the new function hasn't been run in during setup of this test so `font` is not longer a registered mime type.

Reply:
Resolved in https://github.com/WordPress/wordpress-develop/pull/2622/commits/22b868c6a5a98537311fbb6bf358e68a60c18303. This occurred to me over the weekend, where I had one of those 🤦‍♀️ moments 🤣